README.md

```diff
@@ -17,6 +17,10 @@ The user can:
 - Choose the background of the badges and upload the logo and the background image
 - Upload a CSV file or manually enter CSV data as: name, type of attendee, designation, nick/handle, organization/project
 
+##Demo
```

Comment:
space is required here so that the markdown format can render it.
thanks you! please make the necessary changes and then we a

Reply:
Done.